packages/react-components/react-motion/stories/src/Motion/index.stories.tsx

```diff
@@ -0,0 +1,20 @@
+import type { Meta } from '@storybook/react';
+
+import description from './MotionDescription.md';
+
+export { DisableMotion } from './DisableMotion.stories';
+export { CustomMotion } from './CustomMotion.stories';
+
+export default {
```

Comment:
all these stories seems to actually go against the patterns we agreed - this and others related in this PR should be a `.mdx` rather than ts which doesn't provide any real `components` https://github.com/microsoft/fluentui/pull/32465#issuecomment-2331293982

Reply:
have you seen this comment @mainframev  ? I guess not as you re-requested my review and didnt cover what I was asking ?